src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -238,7 +238,14 @@ export const dropdownSearchReducer = (state: IDropdownSearchState = dropdownSear
     case DropdownSearchActions.select:
       nextOptions = !state.supportSingleCustomOption
         ? selectSingleOption(state.options, action.payload.addedSelectedOption)
-        : removeCustomOptions(selectSingleOption(deselectAllOptions(state.options, true), action.payload.addedSelectedOption), state.supportSingleCustomOption, false);
+        : removeCustomOptions(
+          selectSingleOption(
+            deselectAllOptions(state.options, true),
+            action.payload.addedSelectedOption
+          ),
+          state.supportSingleCustomOption,
+          false
+        );
```

Comment:
I think it easier to grasp which parameter is for which function when formated like this:
```
: removeCustomOptions(
  selectSingleOption(deselectAllOptions(state.options, true), action.payload.addedSelectedOption),
  state.supportSingleCustomOption,
  false
);
```

Reply:
i think the problem with this is the nesting of functions three levels deep. I'll move selectSingleOption in a const and add the const as the first argument of removeCustomOptions